CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Move title edition on the title itself
+- Move record toggle
```

Comment:
The changelog should be modified in the second commit. There is nothin here related to the first commit (add edition icon)